### PR TITLE
Doc'd RemovedInDjangoXXWarning comments in deprecating a feature guide.

### PR DESCRIPTION
**Pull request by sarahboyce**

### CODE REVIEW

File: docs/internals/contributing/writing-code/submitting-patches.txt [hunk 1]
```diff
@@ -201,17 +201,22 @@ level:
...
+    # RemovedInDjangoXXWarning.
     def test_foo_deprecation_warning(self):
         msg = "Expected deprecation message"
         with self.assertWarnsMessage(RemovedInDjangoXXWarning, msg):
             # invoke deprecated behavior
             ...
 
+You should also add a ``RemovedInDjangoXXWarning`` comment above any code which
+will need to be removed or altered for the XX release. This makes it easier to
+remove when deprecation ends.
```
Comment (felixxm) [inline docs/internals/contributing/writing-code/submitting-patches.txt:218]: These kind of comments are not necessary for all code that will be removed when a deprecation ends, it is especially unnecessary for tests of deprecation warnings or tests decorated with `@ignore_warnings()`.

We're trying to add them to hooks added to keep the previous behavior, or standalone items that are unnecessary or unused when a deprecation ends.

Comment (sarahboyce) [reply docs/internals/contributing/writing-code/submitting-patches.txt]: Right you search for `RemovedInDjangoXXWarning` and need to see everything, so the places with warnings already show up, it's the more subtle stuff :+1:

File: docs/internals/contributing/writing-code/submitting-patches.txt [hunk 2]
```diff
@@ -212,6 +212,12 @@ You can also add a test for the deprecation warning::
             # invoke deprecated behavior
             ...
 
+You should add a ``RemovedInDjangoXXWarning`` comment above code which does not
```
Comment (nessita) [inline docs/internals/contributing/writing-code/submitting-patches.txt:215]: In order to reduce the duplication of "you should also add" and "you should add", I would propose the following instead:

`It's important to include a ``RemovedInDjangoXXWarning`` comment above code which does not ...`

And I would recommend adding a small example, this way it's more likely that future contributors would get the diff right the first time they try.

**Review by nessita — COMMENTED** [review]
Thanks! I think this is a valuable adding, I just proposed a small change.

File: docs/internals/contributing/writing-code/submitting-patches.txt (hunk 2 above)
Comment (sarahboyce) [reply docs/internals/contributing/writing-code/submitting-patches.txt]: I found it difficult thinking of a good example, but I gave it a go :+1:

**Review by nessita — APPROVED** [review]
Looks great, thank you both!

**Pull request merged**